### PR TITLE
Adds new integration [atxbyea/samsungrac]

### PR DESCRIPTION
**Pull request by atxbyea**

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "asantaga/wiserHomeAssistantPlatform",
   "asev/homeassistant-uponor",
   "atymic/project_three_zero_ha",
+  "atxbyea/samsungrac",
```
Comment (KTibow) [inline integration:25]:
```suggestion
  "atxbyea/samsungrac",
  "atymic/project_three_zero_ha",
```

**Review by KTibow — CHANGES_REQUESTED** [review]

**Comment by KTibow** [comment]
@atxbyea mind merging my `manifest.json` PR? Thanks!

**Comment by KTibow** [comment]
@atxbyea what services does this register?

**Comment by atxbyea** [comment]
@ludeeus @KTibow is this a bug in the check ?

`::INFO:: Category: integration
::INFO:: Repository: atxbyea/samsungrac
::INFO:: Found hacs.json
::INFO:: hacs.json is valid
Error:  ::error:: ::error:: The homeassistant key in manifest.json is no longer valid`

Says it opens hacs.json but mentions the issue is in manifest.json

According to https://hacs.xyz/docs/publish/start the minimum version supported should be "valid" albeit not "required"

**Comment by ludeeus** [comment]
No, not a bug <https://github.com/atxbyea/samsungrac/blob/3.0/custom_components/climate_ip/manifest.json#L14>

**Pull request merged**